domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -179,6 +180,9 @@ class StateRetriever @Inject constructor() {
       "MultipleChoiceInput" -> InteractionObject.newBuilder()
         .setNonNegativeInt(inputJson.getInt(keyName))
         .build()
+      "ItemSelectionInput" -> InteractionObject.newBuilder()
+        .setSetOfHtmlString(createStringList(inputJson.getInt(keyName)))
```

Comment:
This doesn't seem quite right to me--the key should correspond to an array not an int. See: https://github.com/oppia/oppia-android/pull/270/files#diff-8ee6f925fbb2de9ecb09094ccf9ac5cdR169.

Reply:
Fixed this thanks.